checks/collector.py

```diff
@@ -41,6 +43,18 @@
 FLUSH_LOGGING_INITIAL = 5
 DD_CHECK_TAG = 'dd_check:{0}'
 
+# Description of the format of the `processes` resource check, identical to the legacy check
+# Is sent on the first run of the collector
+PROCESSES_FORMAT_DESCRIPTION = [
```

Comment:
now that we're actually axing the legacy check maybe a description of what each field is about would be helpful, or a pointer to where to find it.